rust/scx_utils/src/topology.rs

```diff
@@ -110,116 +111,41 @@ pub enum CoreType {
 
 #[derive(Debug, Clone, Eq, Hash, Ord, PartialEq, PartialOrd)]
 pub struct Cpu {
-    id: usize,
-    min_freq: usize,
-    max_freq: usize,
-    base_freq: usize,
-    trans_lat_ns: usize,
-    l2_id: usize,
-    l3_id: usize,
-    llc_id: usize,
+    pub id: usize,
+    pub min_freq: usize,
+    pub max_freq: usize,
+    pub base_freq: usize,
+    pub trans_lat_ns: usize,
+    pub l2_id: usize,
+    pub l3_id: usize,
+    pub llc_id: usize,
     pub core_type: CoreType,
 }
 
-impl Cpu {
-    /// Get the ID of this Cpu
```

Comment:
Maybe we should move these comments to the attributes instead of dropping them (rephrasing them a bit, like "ID of this CPU"). Most of these descriptions are trivial but some can be useful to keep.